TLSharp.Core/TelegramClient.cs

```diff
@@ -377,8 +377,15 @@ private void OnUserAuthenticated(TLUser TLUser)
             _session.SessionExpires = int.MaxValue;
 
             _session.Save();
-        }
-
+        }
+
+        public bool IsConnected()

```

Comment:
Why do you make this a method and the other a property? I was expecting some consistency here :)